src/core/hooks/useQueryParamsState.tsx

```diff
@@ -0,0 +1,29 @@
+import * as H from 'history';
+import { DateTime } from 'luxon';
+import { EventTypeEnum } from '../../events/models/Event';
+
+export const defaultSearchParam = '';
+export const defaultDateStartParam = DateTime.local().toISODate();
+export const defaultDateEndParam = DateTime.local()
```

Comment:
These are constants, and should probably be written CAPITALIZED

Reply:
Constants typically use underscores `_` as a delimiter, like so:
``` typescript
const DEFAULT_VALUE = 'foo';
```

Reply:
I knew something was making them unreadable 🤔